### ISSUE TITLE
[RFE] Snapshot retention parameter on destination

### ISSUE THREAD

**Issue by fjcloud** [backube/volsync#265]
**Describe the feature you'd like to have.**
It can be a good thing to have a snapshot retention on destination.

**What is the value to the end user? (why is it a priority?)**
in case that you have a corruption it will replicate it on the only snapshot available.

**How will we know we have a good solution? (acceptance criteria)**
Add a parameter of number of snapshot retention you want on destination.

**Additional context**
I have a big issue on my cluster1 that generate corruption on volumes, it take more than 1 hour to see it, unfortunately volsync has replicated corruption in my cluster2 and I have only one snapshot available on cluster2.

**Comment by JohnStrunk** [comment]
For handling corruption, have you considered [SnapScheduler](https://github.com/backube/snapscheduler)?

**Comment by fjcloud** [comment]
So after we can transfer snapshot with VolSync on a destination cluster ?

**Comment by JohnStrunk** [comment]
No, VolSync only handles PVCs currently. The two could be used in a complementary way though.
SnapScheduler would provide local snapshots to protect against application corruption, allowing quick recovery on the local cluster. VolSync would protect against cluster/namespace-level disasters by mirroring the PVC to a remote cluster.

I'm not suggesting that there's no role for a remote snapshot retention schedule, but the above may get you the protection that you need today.

**Comment by fjcloud** [comment]
Yes thanks for suggestion, will definitely take a look to SnapScheduler!
